app/routes/meetings/publish/uittreksels/index.js

```diff
@@ -28,6 +26,6 @@ export default class MeetingsPublishUittrekselsRoute extends Route.extend(
     if (params.title && params.title.length > 0)
       query['filter[titel]'] = params.title;
 
-    return query;
+    return this.store.query('agendapunt', query);
```

Comment:
pagination is broken here because it's missing logic

Reply:
can you please provide more details here

Reply:
you're not passing the page size and number in params, so pagination is not working

Reply:
fixed, thanks for noticing it